widget/embedded/src/pages/Routes.tsx

```diff
@@ -41,6 +41,7 @@ export function RoutesPage() {
       header={{
         onWallet: () => {
           navigate(wallets_url);
+          updateQuotePartialState('refetchQuote', true);
```

Comment:
I didn't get it. Could you explain it?

Reply:
Also whenever I get back to the `Home` or `Routes` pages the `bestRoute` is called. Is it ok?

Reply:
Whenever a user navigates to the home or routes page, the optimal approach is to trigger a request for the latest quotes, including updated parameters such as source and destination tokens, input amount, swappers, and slippage, to ensure the most current data is displayed. However, there are exceptions to this rule. If a user navigates between the home and routes pages, the request should not be initiated to prevent disrupting and confusing users who have already selected a quote.
A flag is stored in the quotes store to determine whether to initiate the request for quotes or not based on this conditions.